mmrotate/visualization/local_visualizer.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional
+
+import numpy as np
+import torch
+from mmdet.structures.mask import bitmap_to_polygon
+from mmdet.visualization import DetLocalVisualizer
+from mmdet.visualization.palette import _get_adaptive_scales, get_palette
```

Comment:
get_palette may be moved into local_visualizer.py. It's better to directly import from mmdet.visualization.

Reply:
see https://github.com/open-mmlab/mmdetection/pull/8517

Reply:
Waiting for https://github.com/open-mmlab/mmdetection/pull/8517 to be merged.